### PR TITLE
Add TextDecoderStream & TextEncoderStream to GroupData

### PR DESCRIPTION
**Pull request by sideshowbarker**
Fixes https://github.com/mdn/content/issues/12464

### CODE REVIEW

File: files/jsondata/GroupData.json [hunk 1]
```diff
@@ -322,7 +322,9 @@
         "StaticRange",
         "Text",
         "TextDecoder",
+        "TextDecoderStream",
         "TextEncoder",
+        "TextEncoderStream",
```
Comment (wbamberg) [inline files/jsondata/GroupData.json:327]: I'm not sure why these should be in DOM. It's my possibly (faulty understanding) that in the Web/API docs, an interface should belong to only one API, which corresponds to the spec that describes it.

Comment (sideshowbarker) [inline files/jsondata/GroupData.json:327]:
```suggestion
        "TextEncoder",
```

Comment (sideshowbarker) [reply files/jsondata/GroupData.json]: > I'm not sure why these should be in DOM. It's my possibly (faulty understanding) that in the Web/API docs, an interface should belong to only one API, which corresponds to the spec that describes it.

OK, removed the part of the patch which was adding them there

**Review by wbamberg — APPROVED** [review]
👍 thank you!

**Pull request merged**